src/checkBox.js

```diff
@@ -0,0 +1,14 @@
+class CheckBox {
+  constructor(task, list) {
+    this.node = document.createElement('input');
+    this.node.type = 'checkbox';
+    this.node.checked = task.completed;
+
+    this.node.addEventListener('change', () => {
+      task.completed = this.node.checked;
+      list.store();
+    });
+  }
```

Comment:
- [ ] Kindly consider adding `text-decoration: line-through;` property for todo when checked as shown in the [To Do list example with minimalist demo](https://www.youtube.com/watch?v=AcUd-_Yjjqg)

| Your design  | Video design|
| ------------- | ------------- |
|![image](https://user-images.githubusercontent.com/53402600/206717751-21032f36-e797-4750-956c-6009bf75bd66.png) | ![image](https://user-images.githubusercontent.com/53402600/206717830-b61e866f-0c87-4720-91ad-782566363f1b.png) |